### PR TITLE
Fix bugs in load_in_patch script on Heroku

### PR DESCRIPTION
**Pull request by La0**
Found 2 bugs when running on Heroku testing:
1. weirdly i get a decode exception using parsepatch there
2. saw a lot of diffs processed multiple times, my sql request lacked a `DISTINCT`

### CODE REVIEW

**Comment by marco-c** [comment]
> weirdly i get a decode exception using parsepatch there

That's strange, could you file an issue on https://github.com/mozilla/pyo3-parsepatch/issues, with the example case where it is failing.

**Comment by La0** [comment]
Done https://github.com/mozilla/pyo3-parsepatch/issues/15

File: backend/code_review_backend/issues/management/commands/load_in_patch.py [hunk 1]
```diff
@@ -95,7 +95,10 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         # Only apply on diffs with issues that are not already processed
-        diffs = Diff.objects.filter(issues__in_patch__isnull=True).order_by("id")
+        diffs = (
+            Diff.objects.filter(issues__in_patch__isnull=True).order_by("id").distinct()
+        )
+        logger.info("Will process {} diffs".format(diffs.count()))
```
Comment (marco-c) [inline backend/code_review_backend/issues/management/commands/load_in_patch.py]: Nit:
```suggestion
        logger.debug("Will process {} diffs".format(diffs.count()))
```

**Comment by marco-c** [comment]
Could you split the PR in two, since they are two separate bug fixes not depending on each other?

**Comment by La0** [comment]
Ok, this one only has the  `DISTINCT`

**Pull request merged**